main/signers/lattice/Lattice/index.ts

```diff
@@ -355,27 +354,25 @@ export default class Lattice extends Signer {
     const optionalFields = ['gasPrice', 'maxFeePerGas', 'maxPriorityFeePerGas']
 
     optionalFields.forEach(field => {
-      if (field in txJson) {
+      if (field in tx) {
```

Comment:
this is iterating over a different object now and doesn't do the same thing. what was the reason for this change?